### PR TITLE
[#48] Improvement / Update Readme

### LINKED ISSUE #48: Update README

### PR DESCRIPTION
**Pull request by marcocapozzoli**
Resolves #48

### CODE REVIEW

**Review by andre-senna — COMMENTED** [review]
@marcocapozzoli  We'll wait for your in progress changes related to the redesign of DAS constructor before we update the documentation. The way the code is right now, the user would need to access das.server in order to use the remote das and this is not the way it is supposed to be. Users should use only explicit DAS API (methods).

**Comment by andre-senna** [comment]
I believe you can keep this PR open in order to update it when you are done with the changes.

File: README.md [hunk 1]
```diff
@@ -114,9 +117,180 @@ das = DistributedAtomSpace(
...
+
+This way it'ss possible to make queries in your local memory and on the remote machine that you need to pass during the creation of DAS instance. See below:
+
+```python
+from hyperon_das import DistributedAtomSpace
+
+das = DistributedAtomSpace(query_engine='remote', host='192.32.11.45', port=9000)
+```
+
+In the query method you can pass the query_scope parameter specifying whether you want to make the query local or remote. Using "local_only" or "remote_only". If you don't pass the default is remote_only. See bellow:
```
Comment (andre-senna) [inline README.md:218]: You should mention and explain all four possible values for `query_scope`. And explain that one that's not implemented yet thus will raise an exception if selected.

**Review by andre-senna — APPROVED** [review]
Approved, but please make the suggested change before merging.

File: README.md (hunk 1 above)
Comment (marcocapozzoli) [reply README.md]: Fixed. I added the 4 possible types.

**Pull request merged**